src/main/webapp/assets/modules/jobseeker-dashboard/jobseekerSubmissions.js

```diff
@@ -0,0 +1,8 @@
+techlooper.directive("jobseekerSubmissions", function () {
+  return {
+    replace: true,
+    templateUrl: "modules/jobseeker-dashboard/jobseekerSubmissions.html",
+    link: function (scope, el, attrs) {
```

Comment:
'attrs' is defined but never used.<br>'el' is defined but never used.<br>'scope' is defined but never used.